docs/envvars.md

```diff
@@ -298,3 +298,25 @@ Determines if the MDN++ SPA should be reachable or not.
 If the `/api/v1/whoami` does not include a `geo.country` value, fall back on
 this. Setting this allows you to pretend the XHR request to `/api/v1/whoami`
 included this value for `geo.country`.
+
+## Glean (Analytics)
+
+### `REACT_APP_GLEAN_CHANNEL`
+
+**Default: `"stage"`**
+
+- This routes glean analytics to either prod/staging in the backend. See
+  [Glean docs](https://mozilla.github.io/glean/book/reference/general/initializing.html?highlight=channel#gleaninitializeconfiguration)
+
+### `REACT_APP_GLEAN_DEBUG`
+
+**Default: `false`**
+
+- Setting this to true logs glean pings to the console to to help debug.
+
+### `REACT_APP_GLEAN_ENABLED`
+
+**Default: `false`**
+
+- Disables/Enables glean upload. This should be set to `true` in prod and
```

Comment:
Note that flipping things on/off has a defined behaviour: https://mozilla.github.io/glean/book/reference/general/initializing.html#behavior-once-initialized . Might be worth linking to the docs here or stating that flipping this to false after it has been true in production will have consequences (i.e. data deletion)

Reply:
I have added a note about this. Thanks for raising.